cf_cli_java_plugin.go

```diff
@@ -229,7 +245,33 @@ func (c *JavaPlugin) execute(commandExecutor cmd.CommandExecutor, uuidGenerator
 	cfSSHArguments = append(cfSSHArguments, remoteCommand)
 
 	output, err := commandExecutor.Execute(cfSSHArguments)
+	if command == heapDumpCommand {
 
+		finalFile, err := util.FindDumpFile(applicationName, heapdumpFileName)
+		if err == nil && finalFile != "" {
+			heapdumpFileName = finalFile
+		}
+
```

Comment:
When testing it, I was expecting to get some feedback from the plugin as to where it creates the heap dump in the container. This would help to find the heap dump in the container in case `-container-dir` is not provided. It also helps to understand whether the plugin properly detected the file system service path or not.

Maybe it's worth simply adding:
```golang
fmt.Println("successfully created heap dump in application container at: " + heapdumpFileName)
```